### PR TITLE
bt-222: Click on logo redirects to main page

### LINKED ISSUE #222: Click on the BSA logo on every page should lead to the main page

### PR DESCRIPTION
**Pull request by vladjsp**

### CODE REVIEW

File: frontend/src/bundles/common/components/logo/logo.tsx
Comment (nikita-remeslov) [inline frontend/src/bundles/common/components/logo/logo.tsx:1]: what happens if I want to use just the logo? Without link?

**Comment by vladjsp** [comment]
Added a link props to the Logo and fixed Sidebar styles for logo and structure.

File: frontend/src/bundles/common/components/logo/logo.tsx [hunk 1]
```diff
@@ -36,6 +36,29 @@ const Logo = ({
...
+        <Grid container>
+            {link === AppRoute.ROOT ? (
+                <Link
+                    to={link}
+                    className={getValidClassNames(styles.logo, className)}
+                >
+                    <BaseLogo isCollapsed={isCollapsed} />
+                </Link>
+            ) : (
+                <Grid className={getValidClassNames(styles.logo, className)}>
```
Comment (Monyster) [inline frontend/src/bundles/common/components/logo/logo.tsx]:
```suggestion
                <Grid container item className={getValidClassNames(styles.logo, className)}>
```
styles.logo will not be applied if grid is not container

**Review by Monyster — CHANGES_REQUESTED** [review]

File: frontend/src/bundles/common/components/logo/logo.tsx (hunk 1 above)
Comment (vladjsp) [reply frontend/src/bundles/common/components/logo/logo.tsx]: True. I will fix this.

File: frontend/src/bundles/common/components/logo/logo.tsx [hunk 2]
```diff
@@ -32,6 +32,32 @@ const Logo = ({
...
+};
+
+const Logo: React.FC<Properties> = ({
+    className = '',
+    isCollapsed = false,
+    link = { AppRoute },
+}) => {
+    return (
+        <Grid container>
+            {link === AppRoute.ROOT ? (
```
Comment (nikita-remeslov) [inline frontend/src/bundles/common/components/logo/logo.tsx]: this check does not work. it works only for the root page. I think it is better to pass some flag `withLink`.

**Review by nikita-remeslov — CHANGES_REQUESTED** [review]

File: frontend/src/bundles/common/components/logo/logo.tsx [hunk 3]
```diff
@@ -32,6 +33,33 @@ const Logo = ({
...
             )}
+        </>
+    );
+};
+
+const Logo: React.FC<Properties> = ({
+    className = '',
+    isCollapsed = false,
+    withLink = false,
+    link = AppRoute.ROOT,
```
Comment (nikita-remeslov) [inline frontend/src/bundles/common/components/logo/logo.tsx]: we can remove this props

**Review by nikita-remeslov — APPROVED** [review]

**Review by anton-otroshchenko — APPROVED** [review]

**Pull request merged**